sql/core/src/test/scala/org/apache/spark/sql/execution/QueryExecutionSuite.scala

```diff
@@ -220,7 +220,8 @@ class QueryExecutionSuite extends SharedSparkSession {
     assertNoTag(tag5, df.queryExecution.sparkPlan)
   }
 
-  test("Logging plan changes for execution") {
+  // TODO(SPARK-44074): re-enable this test after SPARK-44074 resolved
```

Comment:
This failure is interesting.

Reply:
Yes, interesting.

I found change `SQLConf.PLAN_CHANGE_LOG_LEVEL.key` from `INFO` to `WARN` should be ok. 

But I haven't found the root cause yet